File: SDA/SDA-MVC/ModifiedMVC/src/View/StudentView.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package View;

/**
 *
 * @author fa22-bse-067
 */
// File: StudentView.java
public class StudentView {

    // Method to print student details
    public void printStudentDetails(String name, String regNo, double gpa, String grade) {
        System.out.println("Student Details:");
        System.out.println("Name: " + name);
        System.out.println("Registration Number: " + regNo);
        System.out.println("GPA: " + gpa);
        System.out.println("Grade: " + grade);
    }
}

File: SDA/SDA-MVC/student-app/frontend/src/StudentView.js
// frontend/src/StudentView.js
import React from 'react';

const StudentView = ({ name, regNo, gpa, grade }) => {
    return (
        <div>
            <h3>{name}</h3>
            <p>Registration Number: {regNo}</p>
            <p>GPA: {gpa}</p>
            <p>Grade: {grade}</p>
        </div>
    );
};

export default StudentView;
